### DISCUSSION TITLE
Github Pages opens my read me instead

### DISCUSSION THREAD

**Discussion by Fedozie** [community/community#22678] [Other Feature Feedback, Questions, & Ideas]
<p>I noticed that recently when I created a github page website for my repos, that the code wasn’t compiled to give the design of the website but it instead showed the readme of those files.</p>
<p>I have two repos, we the issue recently;</p>
<ol>
<li><a href="https://github.com/Fedozie/Metre-to-Feet-Converter.git" rel="nofollow noopener">https://github.com/Fedozie/Metre-to-Feet-Converter.git</a></li>
<li><a href="https://github.com/Fedozie/Dicee-Challenge.git" rel="nofollow noopener">https://github.com/Fedozie/Dicee-Challenge.git</a></li>
</ol>
<p>And these are my most recent uploads and I also tried opening them with my netlify account but I still got an error.</p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/fedozie">@Fedozie</a>, welcome to the GitHub Support Community!</p>
<p>For your HTML pages to display at the root of the directory, you’ll want to name them <strong>index.html</strong> (all lowercase). They’re currently served from the file itself:<br>
<a href="https://fedozie.github.io/Metre-to-Feet-Converter/LengthConverter.html" class="onebox" target="_blank" rel="nofollow noopener">https://fedozie.github.io/Metre-to-Feet-Converter/LengthConverter.html</a><br>
<a href="https://fedozie.github.io/Dicee-Challenge/dicee.html" class="onebox" target="_blank" rel="nofollow noopener">https://fedozie.github.io/Dicee-Challenge/dicee.html</a></p>
<p>Renaming <strong>LengthConverter.html</strong> and <strong>dicee.html</strong> to index.html should fix it.</p>
<p>Since they’re purely HTML, I’ll also recommend creating an empty file named <strong>.nojekyll</strong> in the root of the repo to disable jekyll processing.</p>

**Comment by tajmone** [comment]
<aside class="quote no-group" data-username="SethClydesdale" data-post="2" data-topic="129448">
<div class="title">
<div class="quote-controls"></div>
 SethClydesdale:</div>
<blockquote>
<p>Renaming <strong>LengthConverter.html</strong> and <strong>dicee.html</strong> to index.html should fix it.</p>
</blockquote>
</aside>
<p>And if you’d like to preserve the documents’s names in the repository (for end users’ sake) you could then create a copy renamed to <code>index.html</code> of these HTML documents in the <code>docs/</code> subfolder (also copying the CSS and JS assets into it) and change the repository settings for GH Pages to serve website contents from the <code>docs/</code> subfolder instead of the root folder — this way you’ll be able to keep the custom filenames and also show them as the GH Pages home page.</p>
<p>You could add a shell script to carry out the copy and rename operations to automate the process, making it easier to keep the copies in synch with the original files, whenever you update the sources.</p>
<p>Furthermore, if you wish that end users downloading the project via the <kbd>download</kbd> button shouldn’t receive the redundant <code>docs/</code> folder, but only the main files in the project root, you could exclude the folder and its contents from the generated Zip archive by adding an <a href="https://git-scm.com/docs/git-archive#ATTRIBUTES" title="Learn more on Git documentation" rel="nofollow noopener">export-ignore rule</a> in the <code>.gitattributes</code> file:</p>
<pre><code class="lang-auto">docs/** export-ignore
</code></pre>
